charts/opensearch/templates/statefulset.yaml

```diff
@@ -190,6 +190,16 @@ spec:
         secret:
           secretName: {{ .Values.securityConfig.tenantsSecret }}
       {{- end }}
+      {{- if .Values.securityConfig.nodesDnSecret }}
+      - name: nodes-dn
+        secret:
+          secretName: {{ .Values.securityConfig.nodesDnSecret }}
+      {{- end }}
+      {{- if .Values.securityConfig.whitelistSecret }}
+      - name: whitelist
+        secret:
+          secretName: {{ .Values.securityConfig.whitelistSecret }}
+      {{- end }}
```

Comment:
Shouldn't this be allowlist? https://opensearch.org/docs/latest/security-plugin/configuration/yaml/#allowlistyml

Reply:
Yes, it should be. 
I have used 2.0.0 image, in which the `securityadmin.sh` doesn't check for allowlist.yml instead it's still using whitelist.yml.So I guess `securityadmin.sh` need a change before we use allowlist.yml.